spec/consensus/time.md

```diff
@@ -16,15 +16,19 @@ The Time produced by CometBFT satisfies the following properties:
   two block headers `H1` of height `h1` and `H2` of height `h2`,
   it is guaranteed that if `h2 > h1` then `H2.Time > H1.Time`.
 
-- **Byzantine Fault Tolerance**: faulty or malicious processes should not be able
-  to arbitrarily influence (increase or decrease) the block Time.
-  In other words, the Time of blocks should be defined by correct processes.
+- **Byzantine Fault Tolerance**: malicious nodes or nodes with inaccurate clocks should not be able
+  to arbitrarily increase or decrease the block Time.
+  In other words, the Time of blocks should be defined by correct nodes.
 
 In addition, the Time produced by CometBFT is expected, by external observers, to provide:
 
 - **Relation to real time**: block times bear some resemblance to real time.
   In other words, block times should represent, within some reasonable accuracy,
-  the actual time at which blocks were produced.
+  the actual clock time at which blocks were produced.
+  More formally, lets `t` be the clock time at which a block with header `H`
```

Comment:
The use of real time in these specs should be clarified to mean the external source of time to which these nodes synchronize. It is not really real time, but a time reference, since there is no real "real time".

Another way would be to consider the real time the median validator clock. This way it may be used to express better the guarantees of BFT Time as well.

```suggestion
  More formally, lets `t` be the clock time at which a block with header `H`
```

Reply:
Yes, this is odd.